BluetoothAdvertisementsKotlin/app/src/main/java/com/example/bluetoothadvertisements/adapter/ScannerAdapter.kt

```diff
@@ -0,0 +1,78 @@
+/**
+ * Copyright (C) 2021 Google Inc. All Rights Reserved.
```

Comment:
Copyright (C) 2021 The Android Open Source Project

Reply:
ack